src/hdmf/common/resources.py

```diff
@@ -753,130 +846,17 @@ def from_norm_tsv(cls, **kwargs):
                 msg = "Key Index out of range in ObjectKeyTable. Please check for alterations."
                 raise ValueError(msg)
 
+        keys_idx = entity_keys['keys_idx']
+        for idx in keys_idx:
+            if not int(idx) < keys.__len__():
```

Comment:
```suggestion
            if not int(idx) < len(keys):
```

Reply:
same for this and the other uses of __len__ above (sorry I didn't catch these earlier)

Reply:
can you update the other three uses of __len__ in this file?

Reply:
Done